frameworks/AutoGluon/README.md

```diff
@@ -3,3 +3,79 @@
 To run v0.8.2: ```python3 ../automlbenchmark/runbenchmark.py autogluon ...```
 
 To run mainline: ```python3 ../automlbenchmark/runbenchmark.py autogluon:latest ...```
+
+
+## Running with Learning Curves Enabled:
+
+To run with learning curves enabled, you must define a new framework. Instructions on how to do this are listed here:
```

Comment:
`new framework` -> `new framework definition in the frameworks YAML file which enables the appropriate parameters`